README.md

```diff
@@ -1,13 +1,43 @@
 # playwright_framework_tamplate
-## Description
-- webaite
+## About the project
+Automation framework tamplate 
+## Testing strategy
+work in progress..
+## Website description
+Website automationpracice.pl is a clothing store.
 `http://www.automationpractice.pl/index.php`
 ## Install
-`npm install`
+- Install NPM packages
+    ```
+    npm install
+    ```
+- Update     
+    ```
+    npm update
+    ```
+- or 
+    ```
+    npm install -D @playwright/test@latest
+    ```
 ## Run
-### bash
+- run all tests 
+    ```
+    npx playwright test
+    ```
+- run single test headless
+    ```
+    npx playwright test example.spec.ts
+    ```
+- run single test in headed mode
+    ```
+    npx playwright test example.spec.ts --headed
+    ```
+- run chosen file test or files
+    ```
+    npx playwright test tests
+    ```
+### npm scripts
 - run all tests
-`npm run test`
-### powershell
-- run all tests
-`npx playwright test`
\ No newline at end of file
+    ```
+    npm run test
```

Comment:
If we use npm run test, maybe we should also rewrite all commands to run tests for playwright e.g

npm run test --headed etc.

Reply:
i rewrited it

Reply:
rewrote :D